src/core/packager.ts

```
@@ -51,7 +61,17 @@ export const pack = async (
   const processedFiles = await deps.processFiles(safeRawFiles, config, progressCallback);
 
   progressCallback('Generating output...');
-  const output = await deps.generateOutput(rootDir, config, processedFiles, safeFilePaths);
+  let gitMetrics;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)